mpf/config_players/plugin_player.py

```diff
@@ -50,30 +53,36 @@ def register_player_events(self, config, mode=None, priority=0):
 
         self.bcp_client = self._get_bcp_client(config)
 
+        m = self.machine
```

Comment:
The reason I did this was because of line length issues. My linter pointed out a ton of these, and sometimes this was the only way I could resolve it.